docker_files/errors/403.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Erreur 403 (Interdit)</title>
```

Comment:
Ideally this would stay in english and perhaps with a much more minimal styling that doesn't rely on CDNs as much.

Reply:
Sure, I borrow those files from a tutorial, will translate all of this